go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/juju/worker/v3
 
-go 1.19
+go 1.20
```

Comment:
`any` was introduced in 1.18, is there some other change that means we need 1.20 as the compiler?
I know we're *using* 1.20, but usually I would only move this sort of declaration when we actually need something from it.